src/safeposix/net.rs

```diff
@@ -297,14 +349,14 @@ impl NetMetadata {
                 if addr.is_unspecified() {
                     for portuser in userarr.clone() {
                         if portuser.1 <= 1 {
-                            drop(portuser);
+                            let _ = portuser;
```

Comment:
I'm not sure why all of these got added. At one point @justyoyo123 said he added these to get rid of warnings? But that doesn't completely check out. This is all using the newer version of rust so possibly they changed semantics?

Reply:
Yeah I made these changes during debugging to get rid of warnings. This was just the suggested change by the compiler so I followed it at the time to just get rid of the warnings in case they were causing any issues. It is possible the drops could have been deleted but that's not somthing I tested.

Reply:
@yizhuoliang can you try getting rid of all these empty assignments and make sure everything works correctly.